Content.Shared/_RMC14/Xenonids/Paralyzing/XenoParalyzingSlashSystem.cs

```diff
@@ -80,6 +81,13 @@ private void OnXenoParalyzingSlashHit(Entity<XenoActiveParalyzingSlashComponent>
                 continue;
             }
 
+            if (HasComp<SynthComponent>(entity))
```

Comment:
This should probably be its own component in the future but this is fine for now